1/mdav2/example.xyz

```diff
@@ -0,0 +1,54 @@
+3
+2474.58 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526    0.00000036050    0.06860687586    0.00000602986   
+ 0.00000000000   -1.68491667000    1.06733568474    0.00000422607   -0.54433120391    0.40288831982   
+ 0.00000000000    1.68491667000    1.06733568474    0.00000796891   -0.54450913059   -0.40302369528   
+
+3
+2335.23 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526    0.00000024512   -0.00001367203    0.04977988550   
+ 0.00000000000   -1.68491667000    1.06733568474   -0.00000501876    0.56614090151   -0.39507637521   
+ 0.00000000000    1.68491667000    1.06733568474    0.00000963445   -0.56595872755   -0.39496497465   
+
+3
+1853.11 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526   -0.00000041420    0.00000191640   -0.06722251145   
+ 0.00000000000   -1.68491667000    1.06733568474   -0.00001740597    0.41919401813    0.53346734578   
+ 0.00000000000    1.68491667000    1.06733568474    0.00000573069   -0.41915068490    0.53340388950   
+
+3
+1214.10 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526    0.00000133232   -0.00001855002    0.00000065993   
+ 0.00000000000   -1.68491667000    1.06733568474   -0.70436687376    0.00013629558    0.00021862242   
+ 0.00000000000    1.68491667000    1.06733568474    0.70434571116    0.00016186466   -0.00022779826   
+
+3
+1190.81 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526   -0.00001244795    0.04785390360   -0.00000933615   
+ 0.00000000000   -1.68491667000    1.06733568474   -0.00015086684   -0.37970831930   -0.57769090807   
+ 0.00000000000    1.68491667000    1.06733568474    0.00039603693   -0.37976930547    0.57772114558   
+
+3
+1139.75 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526   -0.08364749420   -0.00001496544   -0.00000372926   
+ 0.00000000000   -1.68491667000    1.06733568474    0.66376203546    0.00006846700    0.00010313578   
+ 0.00000000000    1.68491667000    1.06733568474    0.66378439116    0.00006552081   -0.00009942936   
+
+3
+8.48 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526    0.08657516585   -0.15757088496   -0.15231663124   
+ 0.00000000000   -1.68491667000    1.06733568474    0.08655595478   -0.15755439709   -0.15229590533   
+ 0.00000000000    1.68491667000    1.06733568474    0.08655706560   -0.15755887333   -0.15231578971   
+
+3
+-3.26 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526    0.21780339222    0.04371949924    0.07856786344   
+ 0.00000000000   -1.68491667000    1.06733568474    0.21781027260    0.04372046767    0.07856874207   
+ 0.00000000000    1.68491667000    1.06733568474    0.21780989969    0.04372043010    0.07856818593   
+
+3
+-7.85 cm^-1
+ 0.00000000000    0.00000000000   -0.13450369526   -0.02427736883   -0.16965551101    0.16170869668   
+ 0.00000000000   -1.68491667000    1.06733568474   -0.02428414725   -0.16966679269    0.16170974128   
+ 0.00000000000    1.68491667000    1.06733568474   -0.02428352961   -0.16967003848    0.16172736527   
+
```

Comment:
There is no need to copy this or other files to your directory. If you want to import a python file, you can append to the pythonpath using `sys`.